ui/start_ui.py

```diff
@@ -864,7 +864,7 @@ def _init_ui(self):
         self.gr_chat = gr_chat
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description="Start UI", add_help=False)
+    parser = argparse.ArgumentParser(description="Start UI", add_help=True)
```

Comment:
Web UI for finetuning and deploying LLMs

Reply:
How about changing it to `Web UI` as introduced in docs?

Reply:
Revised.